_posts/2019-03-06-cockpit-189.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Cockpit 189
+author: Gundersanne
+date: 2019-3-06
+tags: cockpit linux
+slug: cockpit-189
+category: release
+summary: Cockpit with Machines and Services improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 189.
+
+### Machines: Import existing image when creating VM
+
+Choose an existing disk image when create a new VM.
```

Comment:
This just repeats the title. How about dropping "when creating VM" from the title, and be a little more specific in the description? Something like "You can now choose an existing qcow image when creat*ing* an new VM. This will not run an installer,  but just import it into libvirt." @KKoukiou may have some more details?